secret/secret.go

```diff
@@ -133,19 +135,33 @@ func GenerateSecrets(manifest model.Manifest, secrets *v1.Secret, updates *v1.Se
 	}
 
 	for _, key := range sshKeys {
-		dirty = sshKeyGenerate(secrets.Data, updates.Data, key) || dirty
+		sshKeyGenerate(secrets, updates, key)
 	}
 
-	dirty = generateSSLCerts(secrets, updates) || dirty
+	generateSSLCerts(secrets, updates)
 
 	return
 }
 
-func updateVariable(secrets *v1.Secret, updates *v1.Secret, configVar *model.ConfigurationVariable) (dirty bool) {
+func updateVariable(secrets, updates *v1.Secret, configVar *model.ConfigurationVariable) {
 	name := util.ConvertNameToKey(configVar.Name)
 	if len(secrets.Data[name]) == 0 && len(updates.Data[name]) > 0 {
 		secrets.Data[name] = updates.Data[name]
-		dirty = true
+		util.MarkAsDirty(secrets)
 	}
 	return
```

Comment:
Huh, a lone `return` by itself at the end of a void function doesn't look very useful.